.github/workflows/README.md

```diff
@@ -0,0 +1,87 @@
+# Workflows
+
+# Prefixes and file names
+All workflow names should start with [prefix] which contains details on when it should be run. This makes it easier to see this info from the workflows panel, since from here you cant see the workflows `on` directive.
+Current prefixes include
+* `[on <Targets>]`
+  * runs when the branch targeted changes. Targets are a branch name or `PR`, `Prepare` ,or `Trunk`
+    * `PR` will run against branches that are part of a PR whenever that brach changes. PR's will be un-mergable untill these workflows pass
+    * `Prepare` will run against our `prepare-release-for-<trunk_branch_name` branches. These branches are created automatically by `Release Step 1` and are the first stage in releasing
+    * `Trunk` will run against our trunk branches; `main` and `5.x.x` at time of writing.
+* `[Release <step> | <type>]`
+  * run as part of the release process, where step is an integer implying the sequence, and type is one of `manual` or `automatic`
+* `[Issue]`
+  * run when issues are created/changed
+
+The file name of the workflow should match the name, with the prefix and the rest of the name separated by a _. Therefore a workflow with the name `"[on PR] One App Integration Tests"` should be in a file named `on-pr_one-app-integration-tests.yml`
+
+# Workflow descriptions
+
+## on PR
+These workflows check PR code to ensure they are the quality we expect
+
+### [on PR, Trunk] CodeQL
+Runs CodeQL against the PR code
+
+### [on PR, Trunk] One App Unit and Lint Tests
+Runs OneApp's unit tests and linting tests against the PR code
```

Comment:
Why no space between one and app throughout this doc?

Reply:
Because I think in code, so its always either one-app, oneApp, or OneApp, to me... 

I've fixed this now.